### ISSUE TITLE
Can we use DA between Domain3 and Domain4?

### ISSUE THREAD

**Issue by LinGrayy** [cchen-cc/SFDA-DPL#9]
Hi! I looked into your dataset link and found two Refuge datasets Domain3 and Domain4, can we use domain adaptation between these two domains? Is there any meaning for this? thx a lot!

**Comment by rst1991** [comment]
@LinGrayy The entire 400 images of domain 3 and domain 4 are used as the source dataset for training.

**Comment by LinGrayy** [comment]
@rst1991 No, I think they just use 400 training images of domain3 as source dataset. 
![image](https://user-images.githubusercontent.com/49065934/220838227-8bf4287a-a63e-4e19-b26a-02f2cb36499a.png)

**Comment by rst1991** [comment]
Yes, both training and test images are merged of Domain 3 to form the
source training dataset.

**Comment by LinGrayy** [comment]
What about domain4? They don't use it in this work, do they? @rst1991

**Comment by rst1991** [comment]
yes, they have not used domain 4.

**Comment by LinGrayy** [comment]
Then why did you say they use the 400 of domain 4 as the source dataset for training lol. @rst1991

**Comment by rst1991** [comment]
sorry for the confusion. only 400 images of domain 3 are being used.

**Comment by rst1991** [comment]
@LinGrayy Are you able to plot pseudo-labels? Can you please help with it.

**Comment by cchen-cc** [comment]
@LinGrayy Yes, DA can be performed between Domain 3 and Domain 4 as well.
